docs_src/design/adr/core/0022-UoM.md

```diff
@@ -32,38 +32,45 @@ Therefore, EdgeX chooses not to select or adopt a unit of measure specification,
 
 ### Specifying the Units of Measure
 
-Units of measure allowed by the instance of EdgeX will be specified in a configuration file (in TOML format called `uom.toml` by default).  Note:  the UoM configuration is a separate configuration TOML file (separate from the metadata service configuration file - `configuration.toml`).  
+Units of measure allowed by the instance of EdgeX will be specified in a configuration file (in YAML format called `uom.yaml` by default).  Note:  the UoM configuration is a separate configuration YAML file (separate from the metadata service configuration file - `configuration.yaml`).  
```

Comment:
add an EdgeX 3.0 edgy section to state the UoM definition file is changed to YAML instead of TOML in v3